backend/src/services/aws/lambda/lambda.service.ts

```diff
@@ -0,0 +1,59 @@
+import {
+  LambdaClient,
+  CreateFunctionCommand,
+  CreateFunctionCommandOutput,
+  Architecture,
+  Runtime,
+} from '@aws-sdk/client-lambda';
+import AdmZip from 'adm-zip';
+
+type Constructor = {
+  region: string;
+  credentials: {
+    accessKeyId: string;
+    secretAccessKey: string;
+  };
+  role: string;
+};
+
+class Lambda {
+  #lambdaClient: LambdaClient;
+  #role: string;
+
+  constructor({ region, credentials, role }: Constructor) {
+    this.#lambdaClient = new LambdaClient({
+      region,
+      credentials,
+    });
+    this.#role = role;
+  }
+
+  public async creteFunction({
+    name,
+    sourceCode,
+  }: {
+    name: string;
+    sourceCode: string;
+  }): Promise<CreateFunctionCommandOutput> {
+    const zipArchive = new AdmZip();
+
+    zipArchive.addFile('index.js', Buffer.alloc(sourceCode.length, sourceCode));
```

Comment:
```suggestion
    zipArchive.addFile(LambdaDefaultParam.ROOT_FILE Buffer.alloc(sourceCode.length, sourceCode));
```
create `enum` with all default params for lambda